torchvision/prototype/transforms/_meta.py

```diff
@@ -42,7 +42,7 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
 class ConvertColorSpace(Transform):
     # F.convert_color_space does NOT handle `_Feature`'s in general
-    _transformed_types = (torch.Tensor, features.Image, PIL.Image.Image)
+    _transformed_types = (torch.Tensor, features.Image, PIL.Image.Image)  # datumbox: Should we be using `torch.Tensor` or is_simple_tensor()?
```

Comment:
cc @pmeier / @vfdev-5

Reply:
Yes, this implementation is broken if we pass a bbox, it will try to convert it
```
inpt = BoundingBox(torch.tensor([[1, 2, 3, 4]]), format=BoundingBoxFormat.XYXY, image_size=(32, 32))

t = ConvertColorSpace(ColorSpace.RGB, ColorSpace.GRAY)
out = t(inpt)
# ---> 66     return F.convert_color_space(
#      67         inpt, color_space=self.color_space, old_color_space=self.old_color_space, copy=self.copy
#      68     )
```

Reply:
Fixing it in https://github.com/pytorch/vision/pull/6487

Reply:
Sounds good.